src/internal/util/pipe.ts

```diff
@@ -69,6 +69,11 @@ export function pipe<T, A, B, C, D, E, F, G, H, I>(
   ...fns: UnaryFunction<any, any>[]
 ): UnaryFunction<T, unknown>;
 
+/**
+ * pipe() can be called on one or more functions, each of which can take one argument ("UnaryFunction"), to get back
+ * a function that passes the result of each subsequent UnaryFunction to the next one.  It's an implementation of
+ * what is described at https://typeofnan.dev/how-to-use-pipe-the-pipeline-operator-in-javascript/
```

Comment:
We really don't want to link to third party sites that are beyond our control. Can you please remove this link? 

I think a better add might be to show the primary use case for this utility with RxJS, which is to compose an operator from more than one other operator.

Reply:
I removed the link and improved the description.  I agree that your suggestion is better than the external link, and if you feel strongly about showing the primary use case, it should be an example, rather than in the description.  In the description, it might be confusing in relation to the fundamental behavior of pipe().